Hatnya/Screens/CreateStudy/UIComponent/DayButtonView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  DayButtonView.swift
+//  Hatnya
+//
+//  Created by kelly on 2022/07/21.
+//
+
+import UIKit
+
+class DayButtonView: UIButton {
+    var  isClicked = false
```

Comment:
p1;

아앗... 띄어쓰기가 ...
```suggestion
    var isClicked = false
```

Reply:
어이쿠야! 제거했습니다 ㅎㅎ
5426e2ba3f1063323b7e975f74676d8bc5212eb1